lib/ProductOpener/Export.pm

```diff
@@ -306,21 +333,42 @@ sub export_csv($) {
 
 						if (defined $tags_fields{$field}) {
 							if ((defined $product_ref->{$field . "_tags"}) and (scalar @{$product_ref->{$field . "_tags"}} > 0)) {
-								$populated_fields{$field} = sprintf("%08d", $group_number * 1000 + $item_number);
+								# Export the tags field in the main language of the product
+								$populated_fields{$group_prefix . $field} = sprintf("%08d", $group_number * 1000 + $item_number);
```

Comment:
why compute `sprintf("%08d", $group_number * 1000 + $item_number)` once and use it all over the place there after.

```perl
my $field_id = sprintf("%08d", $group_number * 1000 + $item_number);
...
$populated_fields{$group_prefix . $field} = $field_id;
...
$populated_fields{$group_prefix . $field . "_tags"} = $field_id . "_tags";
```

Reply:
Good point, I'll change it.